devtools/dump_devinfo.py

```diff
@@ -285,10 +292,30 @@ async def _make_requests_or_exit(
             )
         )
         exit(1)
-    except Exception as ex:
+    except SmartDeviceException as ex:
         click.echo(
             click.style(f"Unable to query {name} at once: {ex}", bold=True, fg="red")
         )
+        if (
+            isinstance(ex, TimeoutException)
+            or ex.error_code == SmartErrorCode.SESSION_TIMEOUT_ERROR
+        ):
+            click.echo(
+                click.style(
+                    "Timeout, try reducing the batch size via --batch-size option.",
+                    bold=True,
+                    fg="red",
+                )
+            )
```

Comment:
Would you mind creating a simple `err(msg)` function and convert to use that for errors? Otherwise this looks good to me.

Reply:
Done